src/quo2/components/dividers/divider_line/style.cljs

```diff
@@ -2,10 +2,10 @@
   (:require [quo2.foundations.colors :as colors]))
 
 (defn divider-line
-  [blur? theme]
+  [{:keys [blur? padding-top padding-bottom theme]}]
   {:border-color        (if blur?
                           (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5 theme)
                           (colors/theme-colors colors/neutral-10 colors/neutral-90 theme))
-   :padding-top         12
-   :padding-bottom      8
+   :padding-top         (when padding-top padding-top)
+   :padding-bottom      (when padding-bottom padding-bottom)
```

Comment:
@J-Son89 refactored divider component so padding top / bottom can be customized as it had some hardcoded values. There were no usage of this component as far as I could search (previous to integratin on this PR as you suggested), so this shouldn't affect other screens or components.

Reply:
Nice one, curious though should it be margin instead? Perhaps it's all the same however 🤷‍♂️

Reply:
It think it is the same for the purpose of this view

Reply:
Oh actually, if this is an external prop passed in it's better to just use a 'container-style' prop that gets merged to this style object. It's what we are doing for all other quo2 components.

Reply:
Yep that sounds better, I updated with that approach